tests/custom_metadata-testcase.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * Base unit test class for Custom Metadata
+ */
+class CustomMetadata_TestCase extends WP_UnitTestCase {
+	public function setUp() {
+		parent::setUp();
+
+		global $custom_metadata;
+		$this->_toc = $custom_metadata;
```

Comment:
This suggests it has been copied from the Table of Contents plugin? 

There are currently no tests anyway, so there's no need to create a custom testcase if it's not needed.